### ISSUE TITLE
Change zoomed image margin

### ISSUE THREAD

**Issue by babyPrince** [rpearce/react-medium-image-zoom#139]
Hey there, Thank you for the great component! 

I'm having issue adjusting the maximum space around the zoomed image. Currently is something like 45px top bottom, what would be the way to adjust those margins? I've tried `zoomMargin` but it's making the image really small in the centre of the screen when clicked, using:

```
<ImageZoom
  defaultStyles={{
    overlay: {
      backgroundColor: 'none',
    }
  }}
  zoomMargin="0"
```

Thanks

**Comment by rpearce** [comment]
@babyPrince, you've pointed out a small issue that's not quite intuitive: `String` vs `Number`. Here's a picture of the docs:
<img width="923" alt="image" src="https://user-images.githubusercontent.com/592876/47452921-dc879f80-d827-11e8-8876-abd903620991.png">
As you can see, it's a `Number` type that's expected, but you're sending a string. It should be able t handle both, though. I'll write up an issue, fix it and add you to the [all contributors list](https://github.com/rpearce/react-medium-image-zoom#contributors)!

If you want to fix your issue immediately, change `zoomMargin="0"` to `zoomMargin={0}`.

**Comment by rpearce** [comment]
On second thought, I'll use this issue for issue tracking 🤦‍♂️

**Comment by rpearce** [comment]
@babyPrince the fix has been published in `v3.0.15`, and you have been added to the [all contributors list](https://github.com/rpearce/react-medium-image-zoom#contributors). Thanks for the bug report!

**Comment by babyPrince** [comment]
Hey @rpearce Thank you for the reply. Was reading about the number in the docs, but the whole time didn't hit me I was passing a string!
